### DISCUSSION TITLE
High scores table for each lab

### DISCUSSION THREAD

**Discussion by XCemaXX** [dendibakh/perf-ninja#74] [Q&A] [answered]
Is it possible to view not only a baseline and my solution timings on CI, but also the "high scores table" from other contributors results? Or at least author's results?

**Comment by dendibakh** [comment, marked as answer]
Hello, it is definitely possible, but ...
1. Perf-ninja is not designed as a competitive resource, but rather as a learning platform. If someone wants to compete with others, https://highload.fun is a good option.
2. Having a leaderboard would require maintaining absolute scores (timings in milliseconds, etc.). This will add a maintenance burden as scores will change once I change the system configuration, update the compiler version, etc.

**Reply by XCemaXX** [reply]
Thank you for the highload.fun link. Will try.
I suggested leadearboard because I wanted to see other solutions and find better ways of optimizations. Someone can suggest solution with better time than authors. Yeah, I undestand that it will more maintenance effort.
Competing is not the case;)

**Reply by dendibakh** [reply]
Got it. Yeah, that makes sense. Maybe a discussion page on Github would be a good place to discuss lab assignments.
